code/modules/chemistry/Chemistry-Tools.dm

```diff
@@ -393,10 +393,10 @@ ABSTRACT_TYPE(/obj/item/reagent_containers)
 
 	attack_self(mob/user as mob)
 		if (src.splash_all_contents)
-			boutput(user, "<span class='notice'>You tighten your grip on the [src].</span>")
+			boutput(user, "<span class='notice'>You tighten your grip on the [src]. You will now splash in 10 unit increments.</span>")
```

Comment:
```suggestion
			boutput(user, "<span class='notice'>You tighten your grip on the [src]. You will now splash in [src.amount_per_transfer_from_this] unit increments.</span>")
```

Reply:
"unit increments" feels funny.... "units of the solution" is used to refer to the amount when it is splashed 🤷